.eslintrc.js

```diff
@@ -73,12 +81,10 @@ module.exports = {
     react: {
       version: '18',
     },
-    'import/parsers': {
-      '@typescript-eslint/parser': ['.ts', '.tsx'],
-    },
     'import/resolver': {
       typescript: {
         alwaysTryTypes: true,
+        project: ['./apps/*/tsconfig.json'],
```

Comment:
Now that we specify the projects tsconfigs (and its no longer detected), we should probably add one glob pattern packages so its not unintentionally ignored. 
```suggestion
        project: ['./apps/*/tsconfig.json', ./packages/*/tsconfig.json'],
```